testData/ontologies/flat-apptology.json

```diff
@@ -0,0 +1,18 @@
+{
+    "transform": [
+        {
+            "type": "fluid.transforms.value",
+            "inputPath": "http://registry\\.gpii\\.net/common/magnifierEnabled",
+            "outputPath": "http://registry\\.gpii\\.net/common/magnifierApplication"
+        }, {
+            "type": "fluid.transforms.value",
+            "inputPath": "http://registry\\.gpii\\.net/common/magnification",
+            "outputPath": "http://registry\\.gpii\\.net/common/magnifierApplication"
+        }
+    ],
+    "http://registry\\.gpii\\.net/common/screenReaderApplication": "http://registry\\.gpii\\.net/common/screenReaderTTSEnabled",
+    "http://registry\\.gpii\\.net/common/stickyKeysApplication": "http://registry\\.gpii\\.net/common/stickyKeys",
+    "http://registry\\.gpii\\.net/common/slowKeysApplication": "http://registry\\.gpii\\.net/common/slowKeysEnable",
+    "http://registry\\.gpii\\.net/common/debounceKeysApplication": "http://registry\\.gpii\\.net/common/debounceEnable",
```

Comment:
Yeah - I've been having thoughts about that too.. Probably something that we should discuss in Toronto - or before, if I catch you on skype... In case of your example, ideally, we would launch both, but only enable slowKeys via one of them. The problem is that we wont always know which settings are associated with slowKeys (or stickyKeys/debounceKeys for that matter), since the capability might be declared directly, as opposed to part of the capabilities transformation, in which case we have no knowledge of how to disable it. And even if it was declared in the capabilitiesTransformations, it might be a complex transform (non-invertable) that would make it impossible for us to know how to turn off that feature.

But there are other more complex cases (hence the two commented out tests above) -- we should talk about all of these... I'll give you a ping on skype to arrange it